translations/Translations.md

```diff
@@ -1,14 +1,15 @@
-| <img alt="Čeština" title="Čeština" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/cz.svg" width="22">                                                                                                                                               | [Čeština](README.cs.md)                                              |
+|                                                                                                                                                   | Translated Link                       |
+| ------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------- |
+| <img alt="Français" title="Français" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/fr.svg" width="22">                     | [Français](README.fr.md)              |
+| <img alt="Español" title="Español" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/es.svg" width="22">                       | [Español](README.es.md)               |
+| <img alt="Deutsch" title="Deutsch" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/de.svg" width="22">                       | [Deutsch](README.de.md)               |
+| <img alt="Português (Brasil)" title="Português (Brasil)" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/br.svg" width="22"> | [Português (Brasil)](README.pt_br.md) |
+| <img alt="Polski" title="Polski" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/pl.svg" width="22">                         | [Polski](README.pl.md)                |
+| <img alt="Ukrainian" title="Ukrainian" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ua.svg" width="22">                   | [Українська](./README.ua.md)          |
+| <img alt="Russian" title="Russian" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ru.svg" width="22">                       | [Русский](./README.ru.md)             |
+| <img alt="Italiano" title="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">                     | [Italiano](./README.it.md)            |
+| <img alt="Indian-Telugu" title="India-Telugu" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/in.svg" width="22">            | [Telugu](./README.te.md)              |
+| <img alt="Čeština" title="Čeština" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/cz.svg" width="22">                       | [Čeština](README.cs.md)               |
+| <img alt="SriLankan-Tamil" title="SriLankan-Tamil" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/lk.svg" width="22">       | [SL-Tamil](README.ta.md)              |
```

Comment:
Just to make sure is all good, change the route to `./README.ta.md` (optional)

Reply:
I fixed this. any suggestions??

Reply:
All good, you can merge

Reply:
Thank you for your Review. <3